docs/_includes/styles.html

```diff
@@ -0,0 +1,3 @@
+<!-- <link rel="stylesheet" href="{{site.baseurl}}{{ site.assets_base_url }}css/app.css" /> -->
```

Comment:
Sanity check: should this be here, commented out?

Reply:
I removed Less from the project with a new commit, so it's no longer relevant.